js/blocks/ProgramBlocks.js

```diff
@@ -30,14 +36,39 @@ function setupProgramBlocks(activity) {
             ]);
 
             this.formBlock({
-                //.TRANS: load the heap contents from a URL
```

Comment:
I may have missed some of these previously, but //.TRANS: are notes that get put into the PO files for translation hints. It should stay in place, just above the line with the string inside _( )